code/datums/helper_datums/getrev.dm

```diff
@@ -21,13 +23,63 @@ var/global/datum/getrev/revdata = new()
 	if(testmerge.len)
 		world.log << commit
 		for(var/line in testmerge)
-			if(line)
-				world.log << "Test merge active of PR #[line]"
+			world.log << "Test merge active of PR #[line]"
 		world.log << "Based off master commit [parentcommit]"
 	else
 		world.log << parentcommit
 	world.log << "Current map - [MAP_NAME]" //can't think of anywhere better to put it
 
+/datum/getrev/proc/DownloadPRDetails()
+	if(!text2num(config.githubrepoid))
+		world.log << "Invalid github repo id: [config.githubrepoid]"

```

Comment:
Make this a isnum() (like above) but add a if (!config.githubrepoid) above this if with just a simple return.